### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *blazesportsintel* failed.

### ISSUE THREAD

**Issue by ahump20** [ahump20/BSI#319]
### <span aria-hidden="true">❌</span> Deploy Preview for *blazesportsintel* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 0ab97edcb972f05715e9e2db0d4e412203ba812d |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/blazesportsintel/deploys/690661167f6ba00008e46864 |

_Originally posted by @netlify in https://github.com/ahump20/BSI/pull/314#issuecomment-3476718341_
